docs/usecases/mobility-causal-interventions.md

```diff
@@ -0,0 +1,35 @@
+# Causal Interventions
+
+## Description
+
+We provide another workflow for a digital twin that implements the mobility causal intervention framework to evaluate the robustness of deep learning models towards data distribution shifts, with the application of individual next location prediction [^1].
+
+## Digital Twin Workflow
+
+Overview of the mobility causal intervention workflow in ODTP:
+
+``` mermaid
+graph TB
+    subgraph ODTP
+        postgresql-dataloader --> odtp-mobility-simulation
```

Comment:
Exchange `postgresql-dataloader` to `sql-dataloader`. 

Also add `postgis-dataloader` connection to `odtp-mobility-simulation`

Reply:
you mean like this: 

``` mermaid
graph TB
    subgraph ODTP
        sql-dataloader --> odtp-mobility-simulation
        odtp-mobility-simulation --> odtp-mobility-metrics
        odtp-mobility-simulation --> odtp-next-location-prediction
        postgis-dataloader --> odtp-mobility-simulation
    end    
```